apps/data-types.md

```diff
@@ -652,7 +652,7 @@ This object is used for the attributes of basePrice, depositPrice, ecoParticipat
 
 | Attribute | Type | Description |
 | - | - |  - |
-| productNumber | string | The product number (mandatory). |
+| productNumber | string | The product number (if not passed through, an automatically system generated product number will be used). |
```

Comment:
```suggestion
| productNumber | string | The product number. If no product number is provided, an automatically system generated one will be used). |
```
Could we rephrase it like that? The part in the brackets should be an individual sentence, in my opinion.

Reply:
If you want :)